config/settings/rollover.yml

```diff
@@ -6,3 +6,5 @@ bg_jobs:
     cron: "0 0 * * *" # daily at midnight
     class: "SaveStatisticJob"
     queue: save_statistic
+current_recruitment_cycle_year: 2021
+allocation_cycle_year: 2021
```

Comment:
That is conflicting....
`allocation_cycle_year` should be (`current_recruitment_cycle_year` -1)

This is so that the previous values can be shown